applications/GeoMechanicsApplication/custom_elements/small_strain_U_Pw_diff_order_element.hpp

```diff
@@ -49,6 +49,7 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) SmallStrainUPwDiffOrderElement : pub
                                    std::unique_ptr<StressStatePolicy> pStressStatePolicy)
         : UPwBaseElement(NewId, pGeometry, std::move(pStressStatePolicy))
     {
+        this->SetUpPressureGeometryPointer();
```

Comment:
I don't think the this-> is needed here, since it's a function of the same class (and `SmallStrainUPwDiffOrderElement` is not templated)
```suggestion
        SetUpPressureGeometryPointer();
```

Reply:
Agree (I like to add for the eventual case in which we add a template).